TestResultSummaryService/parsers/BenchmarkParser.js

```diff
@@ -1,10 +1,10 @@
 const Parser = require( './Parser' );
 const BenchmarkMetricRouter = require( './BenchmarkMetricRouter' );
 const BenchmarkMetric = require( './BenchmarkMetric' );
-
+const add = (a, b) => a + b;
 const benchmarkDelimiterRegex = /\*\*\*\*\*\*\*\*\*\* START OF NEW TESTCI BENCHMARK JOB \*\*\*\*\*\*\*\*\*\*[\r\n]/;
 const benchmarkNameRegex = /Benchmark Name: (.*) Benchmark Variant: .*[\r\n]/;
-const benchmarkVariantRegex = /Benchmark Name: .* Benchmark Variant: (.*)[\r\n]/;
+const benchmarkVariantRegex = /Benchmark Name: .* Benchmark Variant: (.*?)['\r\n]/;
```

Comment:
PerfNext has been updated so that the above line would only be outputted once instead of twice. Jenkins will no longer output the command before running it.